test/unit/date/format.js

```diff
@@ -52,15 +53,16 @@ Cldr.load(
 	ptCaGregorian,
 	ruCaGregorian,
 	timeData,
-	weekData
+	weekData,
+	metaZones
 );
 
 cldr = new Cldr( "en" );
 
-QUnit.assert.dateFormat = function( date, pattern, cldr, expected ) {
+QUnit.assert.dateFormat = function( date, pattern, cldr, expected, tzId ) {
```

Comment:
Just to confirm, I added the tzId to the end to avoid the changes to rest of the test cases. if your are ok to change all the test then, i can change all the tests.